vue-app/src/store/index.ts

```diff
@@ -187,8 +204,12 @@ export const mutations = {
     state.showCartPanel = !state.showCartPanel
   },
   [RESTORE_COMMITTED_CART_TO_LOCAL_CART](state) {
+    // Flag the committed items. Useful to identify new items added into the Cart
+    // during the reallocation phase.
+    const commitedCart = state.committedCart.map(item => ({ ...item, isCommited: true }))
```

Comment:
Here is the 2) point I was talking about. This flag is going to be used in the Cart template to set different styles in not committed items.

Reply:
Not used anymore. Now, to know if the item is committed or new, it is being calculated in the CartItems component itself.